ads/improvedigital.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {writeScript, checkData, validateDataExists} from '../src/3p';
+
+const adImproveField = ['width', 'height', 'placement', 'optin', 'keyvalue'];
+const adImproveMandatoryField = ['placement'];
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function improvedigital(global, data)
+{
+  checkData(data, adImproveField);
+  validateDataExists(data, adImproveMandatoryField);
+
+  let url = 'https://ad.360yield.com' +
+      '/adj?' +
+      'p=' + encodeURIComponent(data.placement) +
+      '&w=' + encodeURIComponent(data.width) +
+      '&h=' + encodeURIComponent(data.height) +
+      '&optin=' + encodeURIComponent(data.optin) +
+      '&tz=' + new Date().getTimezoneOffset();
+
+  const value = data.keyvalue;
+  let newData = '';
+  const amps = '&';
+  let validKey = 0;
+
+  if (value && value.length > 0) {
+    const keys = value.split('&');
+    for (let i = 0;i < keys.length;i++)
```

Comment:
Could you please address code-style requests I made earlier about this? A nit, but necessary for merge. There are some spaces missing in the for-loop. It should be:
```
for (let i = 0; i < keys.length; i++) {
  if (!keys[i]) {
    continue;
  }
  ...
}
```
Thanks!